src/main/java/com/quantumsoft/qupathcloud/dao/spec/DicomStoresPathBuilder.java

```diff
@@ -15,26 +15,40 @@
 
 package com.quantumsoft.qupathcloud.dao.spec;
 
+import static com.quantumsoft.qupathcloud.dao.Constants.DATASETS;
+import static com.quantumsoft.qupathcloud.dao.Constants.DICOM_STORES;
+import static com.quantumsoft.qupathcloud.dao.Constants.LOCATIONS;
+import static com.quantumsoft.qupathcloud.dao.Constants.PROJECTS;
+import static com.quantumsoft.qupathcloud.dao.Constants.STAGE;
+
 import com.quantumsoft.qupathcloud.exception.QuPathCloudException;
 
-import static com.quantumsoft.qupathcloud.dao.Constants.*;
+/**
+ * DicomStoresPathBuilder to build a path for access to DICOM Stores in chosen Project, Location and
```

Comment:
DicomStoresPathBuilder builds a path to access DICOM Stores in the specified Project, Location and